docs/JestObjectAPI.md

```diff
@@ -647,13 +647,13 @@ This means, if any timers have been scheduled (but have not yet executed), they
 
 Returns the number of fake timers still left to run.
 
-### `.jest.setSystemTime()`
+### `jest.setSystemTime(now?: number | Date)`
```

Comment:
could you update https://github.com/facebook/jest/blob/master/website/versioned_docs/version-26.0/JestObjectAPI.md as well?

Reply:
I think I have (or at least, I intended to) here: https://github.com/facebook/jest/pull/10169/files#diff-82d75bb4a1a86113540ea33c2bd6f24d

Reply:
Oh, seems I'm blind 😅 Thanks!